### DISCUSSION TITLE
Load adjustment not changing results

### DISCUSSION THREAD

**Discussion by ewhite58** [NREL/REopt-API-Analysis#99] [3 - Load Profile Questions] [answered]
Every time I adjust the load (increase to 115% or 150%) in the load profile section of a run in REopt, it doesn't change the results (cost, sizing of PVs/batteries). I can't find anything in the user manual that reflects the newer version of REopt - can anyone help?

**Comment by adfarth** [comment]
Our testing has indicated that the load adjustment slider is working, but it is possible that you've identified an edge case in which it is not. 

In the Results Comparison table, you should see an original and adjusted average annual energy supplied from the grid. Are these numbers identical for you after using the load adjustment slider? 

![image](https://user-images.githubusercontent.com/16725585/220149929-b40b54ba-1824-4039-962c-1a986c25195d.png)

**Comment by ewhite58** [comment]
Thank you for getting back to me so quickly!

I do see this in the Results Comparison table. Perhaps I'm
misunderstanding, but shouldn't an increased electric load mean higher
costs and larger PV and battery sizing in the outputs? I see no changes in
the recommended sizing results section, and I couldn't find any
documentation for the most recent version of REopt that explains how the
load adjustment affects the results.

**Reply by adfarth** [reply]
Apologies that the load adjustment slider is not included in documentation as of yet. The slider simply adjusts the annual electric load that is assumed for the site. Typically, we would expect that a larger electric load should increase PV and battery sizing, but it is possible that this isn't the case for your specific analysis. Would you be willing to share the run-uuid for this analysis (or the results page URL) so we can investigate?

**Comment by ewhite58** [comment]
Yes!

Here is the 100% electric load analysis:
https://reopt.nrel.gov/tool/results/40a2d9ab-b2ad-4221-a96d-164cdb944e1d
And here is the 150% electric load analysis:
https://reopt.nrel.gov/tool/results/720814fe-5c9a-4fd3-9a7e-dbd4a612c853

Please let me know if you need any more information. Thanks so much!

**Reply by adfarth** [reply]
It appears that the load used for both the 100% and 150% cases is showing as 12,679,499 kWh. Is the custom load profile uploaded for each of these analyses identical? If the custom load profile indeed sums to 8,452,999 kWh, could you try running the 100% case again, with re-uploading the load csv?

**Comment by ewhite58** [comment]
The custom load profile summing to 12,679,499 is the 100% case, as I did
the math from utility bills to confirm. The load profile for each of the
analyses are identical, and rerunning the 100% case with a reuploaded csv
gave me the same results. I would expect the 150% case to have about
19,000,000 kWh.

**Comment by ewhite58** [comment]
Hi all,

Just wanted to check in and see if there was any progress on this issue.

Best,
Eva

**Comment by adfarth** [comment, marked as answer]
Hello, we have identified and now fixed a bug in the web tool in which the load adjustment slider bar was incorrectly adjusting a custom load profile. The slider bar should now work as expected (multiply the custom load in each timestep by XXX%). Apologies for the inconvenience and thank you for bringing this to our attention!
